pyVim/connect.py

```diff
@@ -804,6 +834,37 @@ def SmartConnect(protocol='https', host='localhost', port=443, user='root', pwd=
                   b64token=b64token,
                   mechanism=mechanism)
 
+def SmartConnectNoSSL(protocol='https', host='localhost', port=443, user='root', pwd='',
+                      service="hostd", path="/sdk",
+                      preferredApiVersions=None, keyFile=None, certFile=None,
+                      thumbprint=None, sslContext=None, b64token=None, mechanism='userpass'):
```

Comment:
I wondered about this but wanted to maintain consistency with the main Connect functions